contracts/external/actions/VaultAccountAction.sol

```diff
@@ -63,6 +63,10 @@ contract VaultAccountAction is ActionGuards, IVaultAccountAction {
             strategyTokens = vaultAccount.settleVaultAccount(vaultConfig, block.timestamp);
         }
 
+        // If establishing a new vault account, fCash must be borrowed. An account cannot simply deposit
+        // into a vault without establishing some borrow size.
+        if (vaultAccount.maturity == 0) require(fCash > 0, "Must Borrow");
```

Comment:
If we check your collateral ratio on enter, exit, and roll anyway, isn't this requirement superfluous? If the account doesn't borrow then the collateral ratio check will fail and the transaction will revert right

Reply:
The maxAccountRequiredCollateralRatio may be unset which would bypass the other requirement and therefore you can enter with 0 fCash.

Reply:
Ok to me this feels a little hacky. If it's a problem that you enter the vault without borrowing then presumably we would put in a maxAccountRequiredCollateralRatio right? I guess I don't see why it would be the case that we would want to institute this restriction without setting in a max ratio